google-cloud-storage/src/main/java/com/google/cloud/storage/Retrying.java

```diff
@@ -115,6 +120,14 @@ static <T, U> U run(
     }
   }
 
+  @NonNull
+  static GrpcCallContext newCallContext() {
```

Comment:
Does this need a comment or more descriptive name clarifying that it adds the idempotency header? I think it could be confusing otherwise.

Reply:
Since this method is attached to the `Retrying` class, I feel it carries the context with it that the `GrpcCallContext` returned from this method contains things related to retries. I think the idempotency token is an implementation detail of what goes into the context.